tools/gluten-it/common/src/main/scala/io/glutenproject/integration/tpc/Constants.scala

```diff
@@ -43,6 +43,8 @@ object Constants {
     .set("spark.sql.parquet.enableVectorizedReader", "true")
     .set("spark.plugins", "io.glutenproject.GlutenPlugin")
     .set("spark.shuffle.manager", "org.apache.spark.shuffle.gluten.celeborn.CelebornShuffleManager")
+    .set("spark.driver.extraClassPath", "/opt/hadoop/*")
+    .set("spark.executor.extraClassPath", "/opt/hadoop/*")
```

Comment:
Can we insteadly add gluten-it options, for example, `--extra-conf=spark.driver.extraClassPath=/opt/hadoop/*` to velox_be.yml?

Reply:
@zhztheplayer done.